src/app/store.ts

```diff
@@ -1,10 +1,19 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
 
-const rootReducer = combineSlices();
+import todosSlice from '../features/todos';
+import filterSlice from '../features/filter';
+import currentTodoSlice from '../features/currentTodo';
+
+const rootReducer = combineSlices({
+  todos: todosSlice,
+  filter: filterSlice,
+  currentTodo: currentTodoSlice,
+});
```

Comment:
Replace `combineSlices` with `combineReducers` to properly combine the slices into a root reducer. `combineSlices` is not a valid function in `@reduxjs/toolkit`.